BugsnagNetworkRequestPlugin/BugsnagNetworkRequestPlugin/BSGURLSessionTracingDelegate.m

```diff
@@ -97,9 +94,20 @@ - (void)URLSession:(__unused NSURLSession *)session task:(NSURLSessionTask *)tas
 
         NSMutableDictionary *metadata = [NSMutableDictionary dictionary];
         metadata[@"duration"] = @((unsigned)(metrics.taskInterval.duration * 1000));
-        metadata[@"method"] = req.HTTPMethod;
-        metadata[@"url"] = [BSGURLSessionTracingDelegate URLStringWithoutQueryForComponents:urlComponents];
-        metadata[@"urlParams"] = [BSGURLSessionTracingDelegate urlParamsForQueryItems:urlComponents.queryItems];
+
+        NSURLRequest *req = task.originalRequest;
+        if (req == nil) {
+            req = task.currentRequest;
+        }
+        if (req) {
```

Comment:
If there's no request (and hence no url) is there any point leaving a breadcrumb at all?

@kattrali

Reply:
Better some info than none?